tutorials/performance/using_servers.rst

```diff
@@ -119,6 +119,32 @@ This is an example of how to create a sprite from code and move it using the low
         var xform = Transform2D().rotated(deg_to_rad(45)).translated(Vector2(20, 30))
         RenderingServer.canvas_item_set_transform(ci_rid, xform)
 
+ .. code-tab:: csharp
+
+    public partial class MyNode2D : Node2D
+    {
+        // RenderingServer expects references to be kept around. GC will otherwise remove your objects sooner or later...
+        Texture2D texture;
```

Comment:
The access modifier should be explicit, our naming convention for private members is camelCase prefixed by an underscore (`_`).

```suggestion
        private Texture2D _texture;
```